packages/generate-coa-pdf-template/project.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "generate-coa-pdf-template",
+  "$schema": "../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "packages/generate-coa-pdf-template/src",
+  "projectType": "library",
+  "targets": {
+    "build": {
+      "executor": "@nx/esbuild:esbuild",
+      "outputs": ["{options.outputPath}"],
+      "defaultConfiguration": "production",
+      "options": {
+        "outputPath": "packages/generate-coa-pdf-template/dist",
+        "outputFileName": "generateContent",
+        "main": "packages/generate-coa-pdf-template/src/generateContent.ts",
+        "tsConfig": "packages/generate-coa-pdf-template/tsconfig.lib.json",
+        "thirdParty": true,
+        "format": ["cjs"],
+        "generatePackageJson": false
+      },
+      "configurations": {
+        "development": {
+          "minify": false
+        },
+        "production": {
+          "minify": true
+        }
+      }
+    },
+    "publish": {
+      "command": "node tools/scripts/publish.mjs generate-coa-pdf-template {args.ver} {args.tag}",
+      "dependsOn": ["build"]
+    },
```

Comment:
publish is not needed for this project, `generate-en10168-pdf-template` and `generate-pdf-template-helpers` as well